THIRD_PARTY_NOTICES

```diff
@@ -131,15 +131,15 @@ This project incorporates components from the projects listed below. The origina
 124.  @bpmn-io/element-templates-validator@0.11.0 (https://github.com/bpmn-io/element-templates-validator)
 125.  @bpmn-io/extract-process-variables@0.6.0 (https://github.com/bpmn-io/extract-process-variables)
 126.  @bpmn-io/feel-editor@0.4.1 (https://github.com/bpmn-io/feel-editor)
-127.  @bpmn-io/properties-panel@0.23.0 (https://github.com/bpmn-io/properties-panel)
+127.  @bpmn-io/properties-panel@0.24.0 (https://github.com/bpmn-io/properties-panel)
```

Comment:
Is this not updated automatically with the release?

Reply:
Yes. It is an automatic update commit, in fact.